src/arm/windows/init.c

```diff
@@ -172,13 +172,13 @@ static bool read_registry(
 	if (*textBuffer) {
 		HeapFree(heap, 0, *textBuffer);
 	}
-	*textBuffer = HeapAlloc(heap, HEAP_ZERO_MEMORY, dataSize);
+	*textBuffer = HeapAlloc(heap, HEAP_ZERO_MEMORY, dataSize << 1);

```

Comment:
Don't use shift, multiply by `sizeof(wchar_t)` instead

Reply:
Addressed in next revision